### ISSUE TITLE
Chapter 1 Question 6 part h typo

### ISSUE THREAD

**Issue by mehstruslehpy** [gaurangsaini/sipser-computation-3rd-solutions#3]
The final state on the right in the image has no 1 transition. The 0 transition for that state should be 0 and 1 instead of just 0. For example consider the computation of 1111 with your current diagram to see the issue I'm talking about.

From: https://github.com/gaurangsaini/sipser-computation-3rd-solutions/blob/master/Chapter%201/6.pdf

**Comment by dimarcom** [comment]
I agree (was going to say the same thing). But thanks for the great work !
